dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/AbstractEventService.java

```diff
@@ -1656,16 +1765,17 @@ private ImportSummary saveEvent( Program program, ProgramInstance programInstanc
                 programStageInstance = createProgramStageInstance( event, programStage, programInstance,
                     organisationUnit, dueDate, executionDate, event.getStatus().getValue(),
                     completedBy, storedBy, event.getEvent(), aoc, importOptions );
+                
+                saveTrackedEntityComment( programStageInstance, event, storedBy );
```

Comment:
good point. the reason I wanted to have saveTrackedEntityComment first is because I wanted to avoid *unnecessarily* updating PSI since both add/create and update can take care of PSI updating.